pkg/executor/util/utils.go

```diff
@@ -66,6 +67,44 @@ func NewStandardExecutors(
 	return executors, nil
 }
 
+func NewLocalStandardExecutors(
+	cm *system.CleanupManager,
+	ipfsMultiAddress,
+	dockerID string,
+) (*local.Local, error) {
+	// Don't allow user to choose the fuse driver in case it has security issues.
+	// ipfsFuseStorage, err := fusedocker.NewStorageProvider(cm, ipfsMultiAddress)
+	// if err != nil {
+	// 	return nil, err
+	// }
+
+	ipfsAPICopyStorage, err := apicopy.NewStorageProvider(cm, ipfsMultiAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	urlDownloadStorage, err := urldownload.NewStorageProvider(cm)
+	if err != nil {
+		return nil, err
+	}
+
+	exDocker, err := local.NewExecutor(cm, dockerID,
+		map[storage.StorageSourceType]storage.StorageProvider{
+			// fuse driver is disabled so that - in case it poses a security
+			// risk - arbitrary users can't request it
+			// storage.IPFS_FUSE_DOCKER: ipfsFuseStorage,
+			storage.StorageSourceIPFS: ipfsAPICopyStorage,
+			// we make the copy driver the "default" storage driver for docker
+			// users have to specify the fuse driver explicitly
+			storage.StorageSourceURLDownload: urlDownloadStorage,
+		})
+	if err != nil {
+		return nil, err
+	}
+
+	return exDocker, nil
```

Comment:
May be a philosophical question. To keep its signature consistent with the rest of the `New*Executors`, shouldn't this func return `map[executor.EngineType]executor.Executor` type instead of `*local.Local`?

Reply:
or maybe I'm just missing something. Dunno

Reply:
Given the similarity betweem `NewLocalStandardExecutors` and `NewStandardExecutors` (diff here -> https://www.diffchecker.com/O4I1R3MN).
There may be a way of collapsing these two and add an if statement?

Reply:
I think that's because It's just using the local docker executor
in the current model that we have is client-server in the network as well as devstack
the local executor just uses the jobspec instead of the whole job since we get the job as a response when we submit a job, so that wouldn't work in this case there are no API ports etc
the job also has deals and other things so that would'nt work in this case